### PR TITLE
Plat 855 add pendingtranscript

### PR DESCRIPTION
**Pull request by IgorMarnat**

### CODE REVIEW

File: ledger-core/virtual/integration/deduplication_test.go [hunk 1]
```diff
@@ -675,12 +675,24 @@ func (s *deduplicateMethodUsingPrevVETest) confirmPending(
 func (s *deduplicateMethodUsingPrevVETest) finishPending(
 	ctx context.Context,
 ) {
+	objectRef := s.server.RandomGlobalWithPulse()
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/deduplication_test.go:678]: `s.GetObject()`

File: ledger-core/virtual/integration/deduplication_test.go [hunk 2]
```diff
@@ -675,12 +675,24 @@ func (s *deduplicateMethodUsingPrevVETest) confirmPending(
...
+	deactivateRequest := utils.GenerateVCallRequestMethod(s.server)
+	deactivateRequest.Callee.Set(objectRef)
+	deactivateRequest.CallSiteMethod = "Destroy"
+	outgoing := s.server.BuildRandomOutgoingWithPulse()
+	deactivateRequest.CallOutgoing.Set(outgoing)
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/deduplication_test.go:683]: `s.pendingOutgoing`

File: ledger-core/virtual/integration/memory_request_test.go [hunk 3]
```diff
@@ -233,12 +233,23 @@ func pendingPrecondition(s *memoryCacheTest, ctx context.Context, t *testing.T)
 		commonTestUtils.WaitSignalsTimed(t, 10*time.Second, s.server.Journal.WaitAllAsyncCallsDone())
 	}
 	{ // send delegation request finished with new state
+		objectRef           := s.server.RandomGlobalWithPulse()
+		deactivateRequest := utils.GenerateVCallRequestMethod(s.server)
+		deactivateRequest.Callee.Set(objectRef)
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/memory_request_test.go:238]: `s.object`

File: ledger-core/virtual/integration/utils/transcript.go [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+	t := &rms.Transcript_TranscriptEntryIncomingRequest{
+		Request: request,
+	}
+	if stateBefore != nil {
+		t.ObjectMemory.Set(stateBefore)
+	}
+	transcript.Entries[0].Set(t)
+	result := &rms.Transcript_TranscriptEntryIncomingResult{
+		ObjectState: rms.NewReference(stateAfter),
+		Reason:      request.CallOutgoing,
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/utils/transcript.go:19]: add `IncomingResult`, please  
just random ref with pulse for now

File: ledger-core/virtual/integration/utils/transcript.go [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/reference"
+	"github.com/insolar/assured-ledger/ledger-core/rms"
+)
+
+func BuildIncomingTranscript(request rms.VCallRequest, stateBefore, stateAfter reference.Holder) rms.Transcript {
+	transcript := rms.Transcript{Entries: []rms.Any{{}, {}}}
+	t := &rms.Transcript_TranscriptEntryIncomingRequest{
+		Request: request,
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/utils/transcript.go:11]: add `Incoming`, please
just random ref with pulse for now

File: ledger-core/virtual/integration/deduplication/constructor_test.go [hunk 6]
```diff
@@ -292,6 +292,17 @@ func (test *DeduplicationDifferentPulsesCase) run(t *testing.T) {
 	}
 
 	if test.VDelegatedRequestFinished != nil {
+		objectRef := server.RandomGlobalWithPulse()
+		deactivateRequest := utils.GenerateVCallRequestMethod(server)
+		deactivateRequest.Callee.Set(objectRef)
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/deduplication/constructor_test.go:297]: `Set(object)`

File: ledger-core/virtual/integration/deduplication/constructor_test.go [hunk 7]
```diff
@@ -292,6 +292,17 @@ func (test *DeduplicationDifferentPulsesCase) run(t *testing.T) {
...
+		deactivateRequest.CallSiteMethod = "Destroy"
+		outgoing := server.BuildRandomOutgoingWithPulse()
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/deduplication/constructor_test.go:299]: `outgoing` param is already initialized

File: ledger-core/virtual/integration/deduplication/constructor_test.go [hunk 8]
```diff
@@ -292,6 +292,17 @@ func (test *DeduplicationDifferentPulsesCase) run(t *testing.T) {
...
+		deactivateRequest.CallOutgoing.Set(outgoing)
+
+		prevStateRef := server.RandomRecordOf(objectRef)
+		stateRef := server.RandomRecordOf(objectRef)
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/deduplication/constructor_test.go:303]: use `object`

**Comment by IgorMarnat** [comment]
All done in validation-trunk

**Pull request closed without merging**